api/services/TagPresenter.js

```diff
@@ -113,15 +118,15 @@ export const fetchAndPresentSummary = (community, tag) =>
     CommunityTag.where({community_id: community.id, tag_id: tag.id})
     .fetch(),
     TagFollow.where({community_id: community.id, tag_id: tag.id})
-    .fetchAll(),
-    Search.forPosts({
-      communities: [community.id],
-      tag: tag.id
-    }).fetchAll(),
+    .count(),
+    bookshelf.knex('posts_tags')
+    .join('post_community', 'post_community.post_id', 'posts_tags.post_id')
+    .where({community_id: community.id, tag_id: tag.id})
+    .count(),
     mostActiveMembers(community, tag),
-    (communityTag, tagFollows, posts, activeMembers) => ({
+    (communityTag, followerCount, postCount, activeMembers) => ({
       description: communityTag.get('description'),
-      follower_count: tagFollows.length,
-      post_count: posts.length > 0 ? posts.first().get('total') : 0,
+      follower_count: followerCount,
+      post_count: postCount[0].count,
```

Comment:
you can wrap this in `Number` so the test can check for a number instead of a string